docs/docs/cmd/spo/list/list-view-field-add.md

```diff
@@ -50,8 +53,8 @@ Add field with title _Custom field_ to view with title _All Documents_ of the li
 m365 spo list view field add --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle Documents --viewTitle 'All Documents' --fieldTitle 'Custom field'
 ```
 
-Add field with title _Custom field_ at the position _0_ to view with title _All Documents_ of the list with title _Documents_ located in site _https://contoso.sharepoint.com/sites/project-x_
+Add field with title _Custom field_ at the position _0_ to view with title _All Documents_ of the list with url _/sites/project-x/lists/Events_ located in site _https://contoso.sharepoint.com/sites/project-x_
```

Comment:
It's not very logical to add a view called 'All documents' to an event list. Shall we change this this a bit?

Also: could you add another example where the web-relative URL is used instead of the server-relative URL?